source/MaterialXContrib/MaterialXNode/MaterialXData.cpp

```diff
@@ -107,17 +124,28 @@ void MaterialXData::registerFragments()
 	{
         if (MHWRender::MFragmentManager* fragmentMgr = theRenderer->getFragmentManager())
 		{
-			const bool fragmentExists = (glslFragmentWrapper->getFragmentName().size() > 0)
-                && fragmentMgr->hasFragment(glslFragmentWrapper->getFragmentName().c_str());
+			const bool fragmentExists = (_xmlFragmentWrapper->getFragmentName().size() > 0)
+                && fragmentMgr->hasFragment(_xmlFragmentWrapper->getFragmentName().c_str());
 
 			if (!fragmentExists)
 			{
-				std::stringstream glslStream;
-				glslFragmentWrapper->getDocument(glslStream);
-				const std::string xmlFileName(Plugin::instance().getResourcePath().asString() + "/tiledImage.xml");
-				fragmentName = fragmentMgr->addShadeFragmentFromFile(xmlFileName.c_str(), false);
+                std::stringstream glslStream;
+                _xmlFragmentWrapper->getDocument(glslStream);
+                std::string xmlFileName(Plugin::instance().getResourcePath().asString() + "/standard_surface_default.xml");
```

Comment:
Looks like this will need to be parameterized, perhaps after this PR. I can sync later and make sure that the texture case works.

Reply:
Yes. We want to remove this.